pkg/option/config.go

```diff
@@ -2429,6 +2436,12 @@ func (c *DaemonConfig) Populate() {
 
 	c.populateDevices()
 
+	c.IPAMClusterPoolIPv4RoutingCIDR = viper.GetStringSlice(operatorOption.IPAMOperatorV4CIDR)
+	if nativeCIDR := strings.Join(c.IPAMClusterPoolIPv4RoutingCIDR[:0], ""); nativeCIDR != "" {
```

Comment:
`strings.Join(c.IPAMClusterPoolIPv4RoutingCIDR[:0], "")` will always return an empty string, so that statement is never going to be true.

`c.IPAMClusterPoolIPv4RoutingCIDR[:0]` will always return an empty slice, then joining it with an empty string doesn't do anything.

@soumynathan Can we maybe just use the first element directly?

```suggestion
	// TODO: Allow to set multiple CIDRs, use the entire slice.
	if len(c.IMAPClusterPoolIPv4RoutingCIDR) > 0 {
		c.ipv4NativeRoutingCIDR = cidr.MustParseCIDR(c.IMAPClusterPoolIPv4RoutingCIDR[0])
	}
```

Reply:
Will address your comments.